docs/design/bundle-sources.md

```diff
@@ -0,0 +1,240 @@
+* [RukPak > Sources > Images > Private image registries](https://github.com/operator-framework/rukpak/blob/main/docs/sources/image.md#private-image-registries)
+
+## Creating a file-based catalog
+
+If you do not have a file-based catalog, you must perform the following steps to initialize the catalog.
+
+*Prerequisites*
+
+* `opm` CLI tool
+* Docker or Podman
+
+*Procedure*
+
+1. Create a directory for the catalog by running the following command:
+
+    ```sh
+    mkdir <catalog_dir>
+    ```
+
+1. Create a Dockerfile that can build a catalog image by running the `opm generate dockerfile` command:
+
+    *Example `<catalog_name>-image.Dockerfile`*
+
+    ```sh
+    opm generate dockerfile <catalog_dir>
+    ```
+
+    *TIP:* If do not want to use the default upstream base image, you can specify a different image using the `-i` flag.
+
+    * The Dockerfile must be in the same parent directory as the catalog directory that you created in the previous step:
+        *Example directory structure*
+
+        ```sh
+        .
+        ├── <catalog_dir>
+        └── <catalog_dir>.Dockerfile
+        ```
+
+1. Populate the catalog with the package definition for your Operator by running the `opm init` command:
+
+    ```sh
+    opm init <operator_name> \
+    --default-channel=preview \
+    --description=./README.md \
+    --icon=./operator-icon.svg \
+    --output yaml \
+    > <catalog_dir>/index.yaml
+    ```
+
+    This command generates an `olm.package` declarative config blob in the specified catalog configuration file.
+
+## Adding a plain bundle to your file-based catalog
+
+Currently, the `opm render` command does not support adding plain bundles to catalogs. You must manually add plain bundles to your file-based catalog, as shown in the example below.
+
+*Prerequisites*
+
+* `opm` CLI tool
+* A plain bundle image
+* A file-based catalog
+* Push access to an image registry, such as [Quay](https://quay.io)
+* Docker or Podman
+
+*Procedure*
+
+1. Verify that your catalog's `index.json` or `index.yaml` file is similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {   
+        {
+         "schema": "olm.package",
+         "name": "<operator_name>",
+         "defaultChannel": "preview"
+        }    
+    }
+    ```
+
+1. To create an `olm.bundle` blob, edit your `index.json` or `index.yaml` file, similar to the following example:
+
+    *Example `<catalog_dir>/index.json` file*
+
+    ```json
+    {
+        "schema": "olm.bundle",
+        "name": "plainbundle:test",
+        "package": "<operator_name>",
+        "image": "quay.io/<image_repo>/plainbundle:test",
+        "properties": [
+            {
+                "type": "olm.package",
+                "value": {
+                "packageName": "<operator_name>",
+                "version": "0.0.3",
+                "olm.bundle.mediatype": "plain+v0"
```

Comment:
This is a separate property entry. The properties field should look something like:
```
"properties": [
  {
    "type": "olm.package",
    "value": {
      "packageName": "<operator_name>",
      "version": "0.0.3"
    }
  },
  {
    "type": "olm.bundle.mediatype",
    "value": "plain+v0"
  }
]
```

Reply:
Good catch! Added this over based on the google doc comment but missed verifying it. Fixed it now.